### PR TITLE
Translation examples-multicluster-gke Keep consistent with the latest…

### PR DESCRIPTION
**Pull request by loverto**
… version

### CODE REVIEW

File: content_zh/docs/examples/multicluster/gke/index.md [hunk 1]
```diff
@@ -303,3 +304,26 @@ $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
...
+除了卸载Istio之外，还应该执行以下操作
+[Kubernetes multicluster安装说明](/zh/docs/setup/kubernetes/multicluster-install/)：
+
+1. 删除 Google Cloud 防火墙规则：
+
+    {{< text bash >}}
+    $ gcloud compute firewall-rules delete istio-multicluster-test-pods --quiet
+    {{< /text >}}
+
+1. 从不再用于 Istio 的每个集群中删除 `cluster-admin` 集群角色绑定：
```
Comment (fleeto) [inline content_zh/docs/examples/multicluster/gke/index.md]: clusterrolebinding 应该不用翻译

File: content_zh/docs/examples/multicluster/gke/index.md [hunk 2]
```diff
@@ -303,3 +304,26 @@ $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
...
+
+    {{< text bash >}}
+    $ kubectl delete clusterrolebinding gke-cluster-admin-binding
+    {{< /text >}}
+
+1. 删除不再使用的任何 GKE 群集。以下是远程集群 `cluster-2` 的示例 delete 命令：
```
Comment (fleeto) [inline content_zh/docs/examples/multicluster/gke/index.md]: 群集 => 集群

File: content_zh/docs/examples/multicluster/gke/index.md (hunk 1 above)
Comment (loverto) [reply content_zh/docs/examples/multicluster/gke/index.md]: 这个不翻译，很奇怪啊

File: content_zh/docs/examples/multicluster/gke/index.md [hunk 3]
```diff
@@ -307,22 +307,21 @@ $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
 
 ## 卸载
 
-除了卸载Istio之外，还应该执行以下操作
-[Kubernetes multicluster安装说明](/zh/docs/setup/kubernetes/multicluster-install/)：
+除了卸载 Istio 之外，还应该执行以下操作 [Kubernetes multicluster 安装说明](/zh/docs/setup/kubernetes/multicluster-install/)：
```
Comment (rootsongjc) [inline content_zh/docs/examples/multicluster/gke/index.md]: multicluster 不翻译下吗？

**Comment by rootsongjc** [comment]
@loverto CI failed with the following output.
```
- ./public/docs/tasks/telemetry/servicegraph/index.html
  *  External link http://www.graphviz.org/ failed: got a time out (response code 0)
- ./public/docs/tasks/traffic-management/secure-ingress/index.html
  *  External link http://www.libressl.org failed: got a time out (response code 0)
htmlproofer 3.9.2 | Error:  HTML-Proofer found 2 failures!
LINTING FAILED
Exited with code 1
```
This is not an error, please make a new commit to trigger the CI.

**Pull request merged**